src/compas_rhino/uninstall.py

```diff
@@ -40,9 +40,13 @@ def uninstall(version=None, packages=None):
     """
     version = compas_rhino._check_rhino_version(version)
 
-    # We install COMPAS packages in the scripts folder
-    # instead of directly as IPy module.
-    scripts_path = compas_rhino._get_rhino_scripts_path(version)
+    if version == "8.0":
+        # In Rhino 8 there is no scripts folder
+        installation_path = compas_rhino._get_default_rhino_ironpython_sitepackages_path(version)
+    else:
+        # We install COMPAS packages in the scripts folder
+        # instead of directly as IPy module.
+        installation_path = compas_rhino._get_rhino_scripts_path(version)
```

Comment:
should we perhaps do both?

Reply:
the goal of uninstalling is to remove all traces of packages, so perhaps we should try removing them from all possible locations...